packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -305,18 +295,17 @@ The `dns` section specifies configuration options for the DNS protocol. The DNS
 
 [source,yaml]
 ------------------------------------------------------------------------------
-packetbeat.protocols:
-  dns:
-    ports: [53]
+packetbeat.protocols.dns:
+  ports: [53]
 
-    # include_authorities controls whether or not the dns.authorities field
-    # (authority resource records) is added to messages.
-    # Default: false
-    include_authorities: true
-    # include_additionals controls whether or not the dns.additionals field
-    # (additional resource records) is added to messages.
-    # Default: false
-    include_additionals: true
+  # include_authorities controls whether or not the dns.authorities field
```

Comment:
That's a good question, I'm not sure. I think it most cases we could go without comments as the required explanations usually follow anyway. Would be worth it to have a style guide for docs that we can all follow? Might be worth a discussion at least.